packages/sn-controls-react/src/viewcontrols/EditView.tsx

```diff
@@ -4,34 +4,23 @@
 import Button from '@material-ui/core/Button'
 import Grid from '@material-ui/core/Grid'
 import { Repository } from '@sensenet/client-core'
-import { ControlSchema } from '@sensenet/control-mapper'
 import { ContentType, GenericContent } from '@sensenet/default-content-types'
-import React, { Component, ComponentType, createElement, ReactElement } from 'react'
+import React, { createElement, ReactElement, useState } from 'react'
 import MediaQuery from 'react-responsive'
 import { reactControlMapper } from '../ReactControlMapper'
-import { ReactClientFieldSetting } from '../fieldcontrols/ClientFieldSetting'
 
 /**
  * Interface for EditView properties
  */
 export interface EditViewProps {
   content: ContentType
-  onSubmit?: (id: number, content: GenericContent) => void
```

Comment:
This was kind of a lie in the first place. We return fields that changed not a GenericContent type. People might think that there is a Name, Id, Path field in this object but Id will never be in the changed fields. We should think about this in the future.

Reply:
saveableFields' type is now changed to `Partial<GenericContent>`